agent/engine/docker_task_engine_test.go

```diff
@@ -423,7 +423,7 @@ func TestSteadyStatePoll(t *testing.T) {
 	// Two steady state oks, one stop
 	gomock.InOrder(
 		client.EXPECT().DescribeContainer("containerId").Return(api.ContainerRunning, DockerContainerMetadata{DockerId: "containerId"}).Times(2),
-		client.EXPECT().DescribeContainer("containerId").Return(api.ContainerStopped, DockerContainerMetadata{DockerId: "containerId"}),
+		client.EXPECT().DescribeContainer("containerId").Return(api.ContainerStopped, DockerContainerMetadata{DockerId: "containerId"}).AnyTimes(),
```

Comment:
Sure, I can do that.  It's in a newer version of gomock, so we'll need to update that first before I can address the TODO.